.browserslistrc

```diff
@@ -3,11 +3,10 @@
 >= 1%
 last 1 major version
 not dead
-Chrome >= 45
-Firefox >= 38
-Edge >= 12
-Explorer >= 10
-iOS >= 9
-Safari >= 9
-Android >= 4.4
-Opera >= 30
+Chrome >= 70
+Firefox >= 60
+Edge >= 15.15063
+iOS >= 10
+Safari >= 10
+Android >= 5
+Opera >= 50
```

Comment:
I don't know if we should have Opera here at all. Opera is a Chromium based browser now.

Reply:
I think it's fine to be explicit though? Each browser has different functionality that comes with major versions, so we might want to review some of the milestones there to ensure we're not shooting ourselves in the foot.

Reply:
Yes, but why make an exception for Opera? I mean, we don't do the same for the other Chromium-based browsers.

BTW just thinking out loud here.